### PR TITLE
Episode 6 Show Nodes

### PR DESCRIPTION
**Pull request by sockeqwe**
- Assuming Discussion after the show issue is #49
- Any ideas for additional links?
- Length and Size of mp3 file have to be adjusted in `feed.xml` if you edit this file
- As the list of episodes in README is growing, should we list the episodes in reverse order (latest episode on top of the list)?

### CODE REVIEW

File: feed.rss [hunk 1]
```diff
@@ -307,5 +307,52 @@
 			]]>
 		</itunes:summary>
 	</item>
+	<item>
+		<guid>https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_6/The.Context.episode.6.mp3</guid>
+		<title>Episode 6, Part1: Continuous Integration (CI) and Continuous Delivery (CD) with Fernando Cejas</title>
```
Comment (artem-zinnatullin) [inline feed.rss]: Maybe add that Fernando works @ SoundCloud?

File: feed.rss [hunk 2]
```diff
@@ -307,5 +307,52 @@
...
+		<link>https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_6/The.Context.episode.6.mp3</link>
```
Comment (artem-zinnatullin) [inline feed.rss]: -> `The.Context.episode.6_part1.mp3`?

Comment (artem-zinnatullin) [inline feed.rss]: or `The.Context.episode.6.part1.mp3`

File: feed.rss [hunk 3]
```diff
@@ -307,5 +307,52 @@
...
+		<itunes:duration>42:39</itunes:duration>
+		<pubDate>Thu, 18 August 2016 7:00:00 EST</pubDate>
```
Comment (artem-zinnatullin) [inline feed.rss]: `Mon, 22 August 2016 7:00:00 EST`

File: feed.rss [hunk 4]
```diff
@@ -307,5 +307,52 @@
...
+		<enclosure url="https://artemzin.com/static/thecontext/episodes/The.Context.episode.6.mp3" length="41154799" type="audio/mpeg"/>
```
Comment (artem-zinnatullin) [inline feed.rss]: url fix

File: show_notes/Episode_6.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+In this episode we have talked about Continuous Integration and Continuous Delivery. We have covered:
+
+ - Continuous Integration
+ - Continuous Delivery
+ - Pair Programming
+ - Release Train
+ - Soundcloud's CI & CD Pipeline
+ - Testing
+
```
Comment (artem-zinnatullin) [inline show_notes/Episode_6.md]: Everything and life lol :D

**Comment by artem-zinnatullin** [comment]
>As the list of episodes in README is growing, should we list the episodes in reverse order (latest episode on top of the list)?

Good idea! Let's do it after releasing this episode

File: feed.rss (hunk 4 above)
Comment (artem-zinnatullin) [inline feed.rss]: new length is `62473519`

File: feed.rss [hunk 6]
```diff
@@ -307,5 +307,52 @@
 			]]>
 		</itunes:summary>
 	</item>
+	<item>
+		<guid>https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_6/The.Context.episode.6.mp3</guid>
+		<title>Episode 6, Part1: Continuous Integration (CI) and Continuous Delivery (CD) with Fernando Cejas</title>
+		<link>https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_6/The.Context.episode.6.mp3</link>
+		<itunes:duration>42:39</itunes:duration>
```
Comment (artem-zinnatullin) [inline feed.rss]: 43:14 now

File: feed.rss [hunk 7]
```diff
@@ -307,5 +307,52 @@
...
+			<ul>
+			<li>Artem Zinnatullin <a href="https://twitter.com/artem_zin">@artem_zin</a>, <a href="http://artemzin.com">personal blog</a>, <a href="https://github.com/artem-zinnatullin">GitHub</a></li>
+			<li>Hannes Dorfmann <a href="https://twitter.com/sockeqwe">@sockeqwe</a>, <a href="http://hannesdorfmann.com">personal blog</a>, <a href="https://github.com/sockeqwe">GitHub</a></li>
+			</ul>
+
+			<h4>Additional links:</h4>
+
+			<ul>
+			<li><a href="https://speakerdeck.com/android10/it-is-about-philosophy-culture-of-a-good-programmer-second-edition">Slides from Fernando's talk at MobileOptimized'2016 </a></li>
+			</ul>
```
Comment (artem-zinnatullin) [inline feed.rss]: https://github.com/android10/Android-CleanArchitecture !!

File: show_notes/Episode_6.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+Fernando Cejas [@hitherejoe](https://twitter.com/fernando_cejas), [personal blog](http://fernandocejas.com), [GitHub](https://github.com/android10)
+
+#### Hosts:
+
+  - Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](http://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+  - Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+
+#### Additional links:
+
+ - [Slides from Fernando's talk at MobileOptimized'2016 ](https://speakerdeck.com/android10/it-is-about-philosophy-culture-of-a-good-programmer-second-edition)
```
Comment (artem-zinnatullin) [inline show_notes/Episode_6.md]: https://github.com/android10/Android-CleanArchitecture

File: feed.rss (hunk 4 above)
Comment (artem-zinnatullin) [inline feed.rss]: https://artemzin.com/static/thecontext/episodes/The.Context.episode.6.part1.mp3

File: feed.rss (hunk 2 above)
Comment (artem-zinnatullin) [inline feed.rss]: https://artemzin.com/static/thecontext/episodes/The.Context.episode.6.part1.mp3

File: feed.rss [hunk 9]
```diff
@@ -307,5 +307,52 @@
 			]]>
 		</itunes:summary>
 	</item>
+	<item>
+		<guid>https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_6/The.Context.episode.6.mp3</guid>
```
Comment (artem-zinnatullin) [inline feed.rss]: https://artemzin.com/static/thecontext/episodes/The.Context.episode.6.part1.mp3

**Comment by artem-zinnatullin** [comment]
@sockeqwe feel free to release it, I'll be driving 800 km and won't be able to do it until tomorrow!

File: feed.rss (hunk 1 above)
Comment (sockeqwe) [inline feed.rss]: Done

File: feed.rss (hunk 3 above)
Comment (sockeqwe) [inline feed.rss]: done

File: feed.rss (hunk 2 above)
Comment (sockeqwe) [inline feed.rss]: done

File: feed.rss (hunk 6 above)
Comment (sockeqwe) [inline feed.rss]: done

**Comment by sockeqwe** [comment]
@artem-zinnatullin Thanks for your Feedback, I have changed all the things ...
So basically merging to master will releas it , right?

**Comment by artem-zinnatullin** [comment]
Yes!

**Comment by artem-zinnatullin** [comment]
Automate everything!

**Pull request merged**